### ISSUE TITLE
PHP using boolean when type is bool

### ISSUE THREAD

**Issue by leon-v** [20Tauri/DoxyDoxygen#149]
Current behaviour:
```
<?
/**
 * { function_description }
 *
 * @param      boolean  $test   The test
 *
 * @return     boolean  ( description_of_the_return_value )
 */
function test(bool $test = false) : bool {
	return $test;
}
?>
```

Expected behaviour:
```<?
/**
 * { function_description }
 *
 * @param      bool  $test   The test
 *
 * @return     bool  ( description_of_the_return_value )
 */
function test(bool $test = false) : bool {
	return $test;
}
?>
```

I recommend changing type boolean to bool.

In PHP `boolean` is a synonym of `bool`, but boolean values `true` and `false` are of type `bool`.

The importance of adhering to `bool` over `boolean` is because:
```
function test(boolean $test = true){ ... }
```
would produce a fatal error when the default is used as the default `true` is `bool` and not `boolean`

References:
https://stackoverflow.com/questions/44009037/php-bool-vs-boolean-type-hinting
https://www.php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration
https://www.php.net/manual/en/language.types.boolean.php

**Comment by 20Tauri** [comment]
Thanks,

In the next release:
   - "bool" will replace "boolean"
   - "int" will replace "integer"

**Comment by leon-v** [comment]
Awesome! had only just realised `integer` was used as well. Was not far off writing up another issue.
Thank you !

**Comment by 20Tauri** [comment]
Now released in version [0.77.0](https://github.com/20Tauri/DoxyDoxygen/releases/tag/0.77.0) (evolution branch)

Requiere `phpdoc_short_primitives` settings to work... Will be fixed in 0.77.1
